doc/sphinx/proofs/writing-proofs/index.rst

```diff
@@ -10,19 +10,17 @@ the user and the assistant.  The building blocks for this dialog are
 tactics which the user will use to represent steps in the proof of a
 theorem.
 
-Incomplete proofs have one or more open (unproven) sub-goals.  Each
-goal has its own context (a set of assumptions that can be used to
-prove the goal).  Tactics can transform goals and contexts.
-Internally, the incomplete proof is represented as a partial proof
-term, with holes for the unproven sub-goals.
+A first section presents the proof mode (the core mechanism of the
+dialog between the user and the proof assistant).  Then, several
+sections describe the available tactics.  One section is dedicated to
+the SSReflect proof language, which proposes a consistent set of
+tactics to replace some of the standard basic tactics.  Finally, the
+last section documents the ``Scheme`` family of commands, which can be
+used to extend the power of the :tacn:`induction` and
+:tacn:`inversion` tactics.
```

Comment:
This would probably be more readable if presented as a bullet list.  Then it wouldn't be necessary to repeat "section" so may times.  The bullet points could include links to the described sections.

Reply:
The bullet point list of sections is just below though.

Reply:
Good point--I only looked at what's in GitHub.  Then I'd suggest adding a little bit of description for each bullet point rather than in a separate paragraph that covers all of them.

Reply:
Unfortunately, this list is auto-generated by Sphinx. I don't know how it can be customized. But another reason for not adding a description for each section just yet is that this chapter is not yet in its final form (the tactics chapter and the detailed examples chapter should go away in favor of multiple new sections). But this will unfortunately have to wait for 8.14.

Reply:
Looks like the `:hidden:` or `:includehidden:` options on `toctree` would let you hide the auto-generated list.  OTOH, if we then provide our own list, we have to remember to update it when we add/remove/rename rst files.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html

Up to you whether you want to try that now or leave it for another day.

Reply:
Let's leave it for later. I think it will make sense to refine the presentation of the sections when the list of sections is final.